modules/profilereview/src/Auth/Process/ProfileReview.php

```diff
@@ -85,7 +85,7 @@ protected function loadValuesFromConfig(array $config, array $attributes): void
      * @param LoggerInterface $logger The logger.
      * @throws Exception
      */
-    public static function validateConfigValue($attribute, $value, $logger)
+    public static function validateConfigValue(string $attribute, mixed $value, LoggerInterface $logger): void
```

Comment:
Have you checked that strong typing won't break any functions?

Reply:
Yes. On your suggestion, I checked over them once more.